BuildTools/.swiftlint-tests.yml

```diff
@@ -51,3 +51,5 @@ excluded:
   - Example/*/${PROJECT_NAME} # App Store Connect SDK structure
   - ${SRCROOT}/Pods
   - ${SRCROOT}/Sources # Folder for open source projects
+  - ${SRCROOT}/*/Sources
```

Comment:
This test config was getting used on too many files; not only tests but also normal source files. And since this test config doesn't have the custom rules, that was triggering the `superfluous_disable_command` warnings.

If we have 2 different SwiftLint config files, we really need to make sure that each config file is used for one set of swift files, with no overlap. I _think_ this does the trick, but these include/exclude lists are a bit of a mess.

Reply:
I resorted to just copying over the custom rules to `.swiftlint-tests.yml`, because this config is still used on many many files that don't belong to tests. I'd love for someone who initially set this up to look over this with me.